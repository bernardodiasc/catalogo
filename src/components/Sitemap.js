import React, { PureComponent } from 'react'

class Sitemap extends PureComponent {
  render() {
    return (
      <footer style={{ display: 'none' }}>
        <nav>
          <ul>
          </ul>
        </nav>
      </footer>
    )
  }
}

export default Sitemap
