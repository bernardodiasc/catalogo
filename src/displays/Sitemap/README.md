# Sitemap

## How to use

```
import Sitemap from 'displays/Sitemap'
```

```
<Sitemap />
```

## Properties

| propName | propType | defaultValue | isRequired |
| - | - | - | - |
