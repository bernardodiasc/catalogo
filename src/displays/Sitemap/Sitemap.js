import React, { PureComponent } from 'react'
import './Sitemap.css'

class Sitemap extends PureComponent {
  static defaultProps = {}

  render() {
    return (
      <footer className="Sitemap">
        <nav>
          <ul>
          </ul>
        </nav>
      </footer>
    )
  }
}

export default Sitemap
