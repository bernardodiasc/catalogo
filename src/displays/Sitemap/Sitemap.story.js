import React from 'react'
import { storiesOf } from '@storybook/react'
import { withKnobs } from '@storybook/addon-knobs/react'
import Docs from 'stories/containers/Docs'

import Sitemap from './Sitemap'
import SitemapReadme from './README.md'
storiesOf('Displays/Sitemap', module)
  .addDecorator(withKnobs)
  .add('Info', () => <Docs>{SitemapReadme}</Docs>)
  .add('Example', () => <Sitemap />)
  .add('Minimal render', () => <Sitemap />)
