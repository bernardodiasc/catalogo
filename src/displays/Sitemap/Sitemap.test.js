import React from 'react'
import { shallow } from 'enzyme'

import Sitemap from './Sitemap.js'

it('renders without crashing', () => {
  const component = shallow(<Sitemap />)
  expect(component).toMatchSnapshot()
})
