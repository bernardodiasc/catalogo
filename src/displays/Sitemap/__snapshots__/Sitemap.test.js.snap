// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<footer
  className="Sitemap"
>
  <nav>
    <ul />
  </nav>
</footer>
`;
